.github/workflows/check.yml

```diff
@@ -14,9 +14,21 @@ jobs:
     container: registry.fedoraproject.org/fedora:latest
     steps:
       - name: Check out repository
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
       - name: Install dependencies
-        run: dnf install -y cargo git-core
+        run: dnf install -y cargo git-core python3-pyyaml
+      - name: Check for unreferenced templates
+        run: |
+          python3 -c 'import yaml; print("\n".join(yaml.safe_load(open("config.yaml"))["templates"]))' |
```

Comment:
When I copy that in a file and run that as a script, I need to add `\` at the end of lines with a continuation on the next line. Is this not needed here?

Reply:
Pipes at the end of a line don't require backslashes for continuations in bash.  Not sure why you're seeing otherwise locally.

Reply:
Scratch that, looks like we don't need `\` here.